### PR TITLE
New updates

### PR DESCRIPTION
**Pull request by prateek45**
I have Implemented Refresh tokens, solved the memory issue.

### CODE REVIEW

File: examples/00_basic/server.py [hunk 1]
```diff
@@ -1,102 +1,172 @@
...
+from functools import wraps
+from passlib.hash import sha256_crypt as sc
+import uuid
+import secrets
+import jwt
+import datetime
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = os.environ.get('Secret')
+secret_key_2 = os.environ.get('Secret_2')
```
Comment (nicbytes) [inline examples/00_basic/server.py:13]: Why are we using a second secret?

File: examples/00_basic/server.py [hunk 2]
```diff
@@ -1,102 +1,172 @@
...
+app.config['SQLALCHEMY _DATABASE_URI'] = 'something'
```
Comment (nicbytes) [inline examples/00_basic/server.py:14]: This should be set to memory: `'sqlite://'`. 
https://docs.sqlalchemy.org/en/14/core/engines.html#sqlite

File: examples/00_basic/server.py [hunk 3]
```diff
@@ -1,102 +1,172 @@
...
+
+db = SQLAlchemy(app)
+blacklist = []
```
Comment (nicbytes) [inline examples/00_basic/server.py:17]: Blacklist is still in memory. This has already been resolved in the main branch.

File: examples/00_basic/server.py [hunk 4]
```diff
@@ -1,102 +1,172 @@
...
+
+@app.route('/',methods=['GET','POST'])
+def index():
+    return 'Welcome to chatbox'
+
+#go to user home
+@app.route('/home/<token><r_token>',methods=['GET','POST'])
+@require_login
+def user_home(current_user,ref_case):
+    if current_user.active == "True":
```
Comment (nicbytes) [inline examples/00_basic/server.py:104]: No variable called current_user

**Review by nicbytes — CHANGES_REQUESTED** [review]
Hey @prateek45 you should merge master into your branch and ensure the code works before making PRs.

Token support has been added in `flask_restful_auth/restful_auth_views.py` (thinking about changing that class/file name to **_flows.py later).

File: examples/00_basic/server.py (hunk 1 above)
Comment (prateek45) [reply examples/00_basic/server.py:13]: No particular reason, i think for better security purposes it is preferred to use different security keys. for example if both the token are leaked and they use same security keys, then the chances of deriving the secret key increases where if we use 2 separate ones, the same reduces

File: examples/00_basic/server.py (hunk 2 above)
Comment (prateek45) [reply examples/00_basic/server.py:14]: oh okay, Actually  I haven't tested the code yet.
I'll correct it.

File: examples/00_basic/server.py (hunk 3 above)
Comment (prateek45) [reply examples/00_basic/server.py:17]: let me check

File: examples/00_basic/server.py (hunk 4 above)
Comment (prateek45) [reply examples/00_basic/server.py:104]: In here the decorator is returning verified user to current user

**Pull request closed without merging**